### PR TITLE
Test comments

### PR DESCRIPTION
**Pull request by martin-ladecky**

### CODE REVIEW

File: .github/workflows/terraform_apply.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+name: 'Terraform Apply on push'
+on:
+  push:
+    branches:
+      - main
+      - test_comments
+#    paths:
```
Comment (d35h) [inline .github/workflows/terraform_apply.yml:7]: Do we need commented code?

File: .github/workflows/terraform_apply.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+#      - 'infrastructure/aws/**'
+env:
+  tf_version: '1.1.6'
+  tf_working_dir: 'infrastructure/aws'
+  tf_actions_working_dir: 'infrastructure/aws'
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+#  TF_VAR_certificate-arn: ${{ secrets.CERTIFICATE_ARN }}
```
Comment (d35h) [inline .github/workflows/terraform_apply.yml:15]: Do we need commented code?

**Comment by martin-ladecky** [comment]
zlý pull request :D

File: .github/workflows/terraform_apply.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+name: 'Terraform Apply on push'
```
Comment (d35h) [inline .github/workflows/terraform_apply.yml:1]: Btw, why `apply` if we're doing plan only?

File: infrastructure/aws/ecs.tf [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  vpc_subnets    = join(",", module.vpc.public_subnets)
+  enable_ssh     = true
+  ssh_sg         = aws_security_group.allow-ssh.id
+  log_group      = "my-log-group"
+  aws_account_id = data.aws_caller_identity.current.account_id
+  aws_region     = var.AWS_REGION
+}
+
+module "my-service" {
+  source              = "github.com/in4it/terraform-modules//modules/ecs-service"
```
Comment (d35h) [inline infrastructure/aws/ecs.tf:19]: Wrong path double-slash
"github.com/in4it/terraform-modules`//`modules/ecs-service"

Comment (d35h) [reply infrastructure/aws/ecs.tf:19]: Also, please review the rest

File: infrastructure/aws/mykey.pub [hunk 5]
```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC4EGvkzpTSP+g5fzUUcBOivETeNu6847aSp0O6y5RkTeJaLgblK6GrQmC7cVdqB3nHxmzfKXvUb44jA+wP7C3dFLAqLRnf6+EPFWcQDLXFar/sZns0/iv3azkrF1TeOvruwlUpnBzumkwtxIs/7RW41Y/tuPcDL8Y52SESv56dqz2nXIj8tbt0+7NO4isUSHYsKeqCkv1CIxSzZA+mLInugvcW7STz+4iUskaCStPlFudEWuId9z4jVyNrBO9DTHaxccy20lbham9O7rFxOXKNX0cSR8Yy2Xbwq+WGPOHGZB8MW+fAs3PreiB6mJcXFpagXimWgqp3e65o1Oc27kXn5zvgIqIk3ZrMlq2Z5iV0seJH3sex046W2kLXsULfD88qp+PjgmPzb22JQ+VKAX6kMF0PpS6x4t8HREEfaAmFuOI8c9UD1SnOD38QPidEVolIimes01boyQ0tVVEdjJ21YKZVbkWkpf1+eps9og9nt7jmTkRNDHOG+Z1DnQldgW8= Martin Ladecky@MartinLadecky
```
Comment (d35h) [inline infrastructure/aws/mykey.pub:1]: Should it be here?

**Pull request closed without merging**